zboxcore/sdk/downloadworker.go

```diff
@@ -93,7 +93,7 @@ func (req *DownloadRequest) downloadBlock(blockNum int64, blockChunksMax int) ([
 	if len(req.encryptedKey) > 0 {
 		encscheme = encryption.NewEncryptionScheme()
 		// TODO: Remove after testing
-		encscheme.Initialize(client.GetClient().Mnemonic)
+		encscheme.Initialize(client.GetClient().Wallet.Mnemonic)
```

Comment:
@Rizary what is the reason of changing this?

Reply:
I'll remove it in the next commit. It is because I want to make the Wallet public. But it turns out already being fixed in the `init()` function.